src/App.js

```diff
@@ -4,16 +4,35 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    userRequest: '',
+  };
+
+  searchFilm = (event) => {
+    this.setState({
+      userRequest: event.target.value,
+    });
+  }
 
   render() {
+    const visibleFilms = moviesFromServer.filter(
+      m => m.title.toLowerCase().includes(
```

Comment:
```suggestion
      movie => movie.title.toLowerCase().includes(
```
Better to use meaningful namings